content/home/3-get-involved.md

```diff
@@ -6,31 +6,31 @@ examples:
   - socialMedia:
       - title: 'Posting on social media to thank the maintainers of their favorite open source projects'
       - image: '/images/thanks.png'
-      - imageDescription: 'Illustration'
+      - imageDescription: 'Illustration by Ariel Davis of a person thanking an open source maintainer on social media'
   - podcasts:
       - title: 'Recording special podcast episodes'
       - image: '/images/podcast.png'
-      - imageDescription: 'Illustration'
+      - imageDescription: 'Illustration by Ariel Davis of a person recording a podcast episode about open source maintainers'
   - meetings:
       - title: 'Organizing special-edition open working group meetings'
       - image: '/images/meetings.png'
-      - imageDescription: 'Illustration'
+      - imageDescription: 'Illustration by Ariel Davis of people at an open working group meeting for open source maintainers' 
   - miniEvents:
       - title: 'Hosting mini-events'
       - image: '/images/events.png'
-      - imageDescription: 'Illustration'
+      - imageDescription: 'Illustration by Ariel Davis of people hosting a mini-event for open source maintainers'
   - summits:
       - title: 'Convening tool-specific maintainer summits'
       - image: '/images/tools.png'
-      - imageDescription: 'Illustration'
+      - imageDescription: 'Illustration by Ariel Davis of people at a tool-specific maintainer summit for open source projects'
   - streaming:
       - title: 'Streaming live on Twitch'
       - image: '/images/twitch.png'
-      - imageDescription: 'Illustration'
+      - imageDescription: 'Illustration by Ariel Davis of a person streaming on Twitch about open source maintainers'
   - donations:
       - title: 'Pledging to donate to support Open Source'
       - image: '/images/donate.png'
-      - imageDescription: 'Illustration'
+      - imageDescription: 'Illustration by Ariel Davis of making a donation to support open source'
```

Comment:
perhaps reword to "Illustration by Ariel Davis of a donation being made to support open source"

Reply:
Hi @abbycabs That's well noted. rewording it in a few